<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestsDialog</class>
 <widget class="QDialog" name="TestsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>rainbow.png</normaloff>rainbow.png</iconset>
  </property>
  <widget class="QPushButton" name="pushButton_appendVerifiableTest">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>86</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Вставить
в таблицу</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_verifiableTests">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>740</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_unverifiableTests">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>405</y>
     <width>740</width>
     <height>190</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_verifiableTests">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>430</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_appendUnVerifiableTest">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>360</y>
     <width>60</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Вставить
в таблицу</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_unverifiableTests">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>375</y>
     <width>430</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_inputNewTest">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_inputNewTest">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>94</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить новый
тест в систему</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_removeSelectedTest">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>116</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить выбранный
тест из системы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>В столбце &quot;Условия&quot;:
&quot;\&quot; - символ новой строки
&quot;+-&quot; будет заменено на &quot;±&quot;
&quot;CC&quot; будет заменено на &quot;°C&quot;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>760</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_noControlParams">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Перечень непроверяемых параметров</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ControlParams">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>75</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Перечень проверяемых параметров</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_removeVerifiableTestFromTable">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>75</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить
выбранный
из таблицы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_removeUnVerifiableTestFromTable">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>350</y>
     <width>70</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить
выбранный
из таблицы</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_verifiableTests_filter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_unverifiableTests_filter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>375</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_verifFilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Фильтр</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_unverifFilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>355</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Фильтр</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
